hack/release-notes.tmpl

```diff
@@ -0,0 +1,20 @@
+# Changelog since {{ .PreviousRevision }}
```

Comment:
I think that this can remain implicit. Changelog is always related to the lust bump, isn't it.